includes/class-urlslab-blocks.php

```diff
@@ -47,6 +49,7 @@ static function init_elementor_blocks() {
 			function ( $manager ) {
 				$manager->register_widget_type( new Urlslab_Related_Articles_Elementor() );
 				$manager->register_widget_type( new Urlslab_Screenshot_Elementor() );
+				$manager->register_widget_type( new Urlslab_TableOfContents() );
```

Comment:
while Elementor version of block isn't implemented, remove pls registration...
otherwise frontend will crash at all:

![Uploading 2023-12-04_18-41-32.png…]()